packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -125,6 +125,8 @@ export const messages = {
     'One or more %s files with the specified path already exist in your workspace. Do you want to overwrite them?',
   warning_prompt_dir_overwrite:
     'A folder with the specified project name already exists in the selected directory. Do you want to overwrite it?',
+  warning_prompt_cmp_file_overwrite:
```

Comment:
@hbc-bindu @ntotten This is what the message currently looks like if the user wants to run a retrieve on a component and he/she already has a local version of that file present. 
![Screen Shot 2019-07-22 at 5 42 58 PM](https://user-images.githubusercontent.com/21326913/61674316-35e20980-aca8-11e9-8c19-6bc6a7dd550a.png)
Thoughts?

Reply:
Hi @AnanyaJha, you could rephrase the first sentence as:
"<Filename>" already exists in your workspace. Do you want to overwrite it?

Reply:
![Screen Shot 2019-07-23 at 2 41 24 PM](https://user-images.githubusercontent.com/21326913/61749566-e65b1680-ad57-11e9-933c-5e54298c1185.png)
 Hi @hbc-bindu I actually tweaked the message a bit, because of some inconsistent behavior we found in the CLI. When the user runs this command, they're expecting the file to be retrieved into their specified default directory. Initially, we were only checking their default directory to see if the file was present- and they would only see this message if it were present.

Occasionally, the file may actually get retrieved into some other package directory instead of the default, potentially overwriting the file from that other package. Now, the user will see this message if the file exists anywhere inside their workspace instead of just in the default package directory. Thoughts?